src/schema.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { Schema } from "./schema";
 import { graphql } from "graphql";
 import 'jest';
+import { persons } from "./schema/data-base/person-database";
```

Comment:
please take data-base folder 1 directory up.
should be import from "./database/person-database";